website/pages/docs/plugins/sub-graph.mdx

```diff
@@ -58,15 +58,15 @@ const combinedSchema = builder.toSchema({ subGraph: ['Internal', 'Public'] });
 
 ### Object and Interface types:
 
-- `defaultSubGraphsForFields`: Default sub-graph for fields of the type to be included in.
+- `defaultForFields`: Default sub-graph for fields of the type to be included in.
```

Comment:
defaultSybGraphsForFields is the correct option here.

Reply:
I get an error in my editor for that:

<img width="1231" alt="image" src="https://user-images.githubusercontent.com/174864/180346833-faf88023-a432-4069-b04d-fa8e9654e8d1.png">

It seems like it's not defined?

https://github.com/hayes/pothos/blob/ab8479e5891a7c0ef6f14b70162a370d1b0f1595/packages/plugin-sub-graph/src/global-types.ts#L57-L63=

Reply:
This is describing the options available on Object and Interface types.  I can't tell from your screenshot where this error is coming from, but it looks (based in the message) like this is for the options based into the builder, not the options for a specific object or interface.

Objects and interfaces have their options extended here: https://github.com/hayes/pothos/blob/main/packages/plugin-sub-graph/src/global-types.ts#L25

Reply:
Whoops, my bad! I'll change it back.